dbx/utils/__init__.py

```diff
@@ -15,7 +18,7 @@ def format_dbx_message(message: Any) -> str:
 def dbx_echo(message: Any):
     formatted_message = format_dbx_message(message)
     try:
-        rich_print(formatted_message)
+        CONSOLE.print(formatted_message, soft_wrap=True)
```

Comment:
it seems that we can also use:

```python
import rich

rich.reconfigure(soft_wrap=True)
rich_print(formatted_message)
```

Reply:
hi @copdips , 
yes, the reconfigure option seems more suitable to me. thanks for pointing it out!